LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2016 Alex Raileanu
```

Comment:
@dreamsurfer The copyright species a name other than mine. Please fix it. Thanks!